tsconfig.json

```diff
@@ -33,7 +33,6 @@
     "dist-schema/**",
     "**/node_modules/**/*",
     "**/third_party/**/*",
-    "packages/angular_devkit/schematics_cli/schematic/files/**/*",
-    "packages/angular_devkit/*/test/**/*"
```

Comment:
Why was the last entry removed? It was meant to exclude the `test` project. Also, I noticed that `modules/testing/builder/projects/**/*` is missing, which would otherwise result in linting the `test` project assets.

Reply:
It caused `ts_project` to not compile tests inside `test/`. Are you saying this was just here for linting?

Reply:
The `ts_project` should exclude tests in the `test/` directory from compilation, as they are "assets" for the actual tests.

Reply:
Done. please check again